api/src/pcapi/routes/e2e/routes/sandbox_data.py

```diff
@@ -0,0 +1,39 @@
+from flask import current_app as app
+
+from pcapi.core.finance import factories as finance_factories
+from pcapi.core.offerers import factories as offerers_factories
+from pcapi.core.offerers.models import VenueTypeCode
+from pcapi.core.users import factories as users_factories
+from pcapi.repository.clean_database import clean_all_database
+from pcapi.sandboxes.scripts.mocks.educational_siren_mocks import MOCK_ADAGE_ELIGIBLE_SIREN
+from pcapi.serialization.decorator import spectree_serialize
+
+
+@app.route("/e2e/pro/create-event-individual-offer", methods=["GET"])
+@spectree_serialize(on_success_status=204)
+def setup_create_event_individual_offer() -> None:
+    clean_all_database()
+    pro_user = users_factories.ProFactory(
+        lastName="PC Test Pro",
+        firstName="97 0",
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏On peut en profiter pour lui donner un vrai nom ? Ou bien il y a un besoin derrière ce choix ?

Reply:
Non on peut même enlever ces champs et laisser les valeurs par défaut de la factory